en/docs/guides/login/session-timeout.md

```diff
@@ -0,0 +1,17 @@
+# Session Timeout 
+
+Follow the steps given below to change the time for which the SSO session can be idle before the user gets logged out. 
+
+{! fragments/session-timeout.md !}
+
+!!! Tip "Configure timeout values globally" 
+    The global configuration can be added in the
+    `<IS_HOME>/repository/conf/deployment.toml` file as shown below.
+    ```
+    [session.timeout]
+    idle_session_timeout= "15m"
```

Comment:
Let's explain the two properties 'idle_session_timeout' and 'remember_me_session_timeout'

Reply:
Modified with a one-line explanation.